pkg/printers/internalversion/printers.go

```diff
@@ -588,6 +588,13 @@ func AddHandlers(h printers.PrintHandler) {
 	}
 	h.TableHandler(storageVersionColumnDefinitions, printStorageVersion)
 	h.TableHandler(storageVersionColumnDefinitions, printStorageVersionList)
+
+	scaleColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "ActualReplicas", Type: "integer", Description: "Observed number of replicas"},
```

Comment:
Should the Description be `autoscalingv1.ScaleStatus{}.SwaggerDoc()["replicas"]`? :thinking: 

ditto for `DesiredReplicas` below?

Reply:
nit: s/Actual/Available to match the deployment type